tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -936,20 +1113,32 @@ export class WebGPUBackend extends KernelBackend {
     if (this.shouldExecuteOnCPU([x])) {
       return this.cpuBackend.neg(x);
     }
-    const program = new UnaryOpProgram(x.shape, unary_op.NEG);
+    const key = this.getHash([webgpu_util.UNARYOPPROGRAM, x.shape,
```

Comment:
Please use a helper function `UnaryOp` like `BinaryOp` did to reduce the key definiation.

Reply:
done.